<?xml version="1.0" encoding="UTF-8"?>
<testsuite errors="0" failures="1" name="matrix_tests.MatrixTest-20170323201128" skipped="0" tests="6" time="0.001">
	<testcase classname="matrix_tests.MatrixTest" name="testChangeRows" time="0.000"/>
	<testcase classname="matrix_tests.MatrixTest" name="testEchelonFormAndReducedEchelonForm" time="0.000"/>
	<testcase classname="matrix_tests.MatrixTest" name="testMultiplyRows" time="0.000"/>
	<testcase classname="matrix_tests.MatrixTest" name="testPrintMatrix" time="0.000"/>
	<testcase classname="matrix_tests.MatrixTest" name="testScalarMultiplication" time="0.000"/>
	<testcase classname="matrix_tests.MatrixTest" name="testMatrixMultiplication" time="0.001">
		<failure message="Lists differ: [3, 0] != [1, 0]

First differing element 0:
3
1

- [3, 0]
?  ^

+ [1, 0]
?  ^
" type="AssertionError">
<![CDATA[Traceback (most recent call last):
  File "/home/hyttijan/MatrixCalculator/src/unittest/python/matrix_tests.py", line 27, in testMatrixMultiplication
    self.assertEqual(newMatrix.matrix[0],[1,0])
AssertionError: Lists differ: [3, 0] != [1, 0]

First differing element 0:
3
1

- [3, 0]
?  ^

+ [1, 0]
?  ^

]]>		</failure>
	</testcase>
	<system-out>
<![CDATA[[ 3  3  -15  9  ]
[ 1  0  -2  1  ]
[ 2  -1  -1  0  ]

]]>	</system-out>
	<system-err>
<![CDATA[]]>	</system-err>
</testsuite>
